tests/KO/private_rhs.dk

```diff
@@ -0,0 +1,5 @@
+#REQUIRE tests.OK.private.
```

Comment:
I would prefer not to add more dk files...

Reply:
Checking retro compatibility might be still relevant since there are problems with the earley one...